testsuite/policy/authorization/auth_policy.py

```diff
@@ -18,13 +18,12 @@ class AuthPolicy(AuthConfig):
     def auth_section(self):
         return self.model.spec.setdefault("rules", {})
```

Comment:
If the rules object is defined as `{}` already in `create_instance()` would it be possible to remove the `setdefault`?
```suggestion
        return self.model.spec.rules
```

Reply:
I left it in case of `rules` key doesn't show in the object at all if it is empty (tell me if I'm wrong and it's not possible). I think it is safer to leave the default here for now

Reply:
I think the `rules` key should always exist if its defined in `model` in `create_instance()` which is run on creation of every `AuthPolicy` object. But the current state is not wrong either, I just think its redundant. 
This PR can be merged without changing this.

Reply:
I run `make test` on your PR with this modification and all tests passed (except the wildcard test error which is caused by something else).